### PR TITLE
feat: support refresh for issues list

### PR DESCRIPTION
**Pull request by GZLiew**
This PR add refreshing behaviour when the `<C-R>` or `F5` key is pressed when viewing issues list

relating to #292 but with no interval implemented

### CODE REVIEW

File: pkg/tui/table.go [hunk 1]
```diff
@@ -165,6 +176,13 @@ func (t *Table) initTable() {
 			}
 		}).
 		SetInputCapture(func(ev *tcell.EventKey) *tcell.EventKey {
+			if ev.Rune() == 'r' {
```
Comment (ankitpokhrel) [inline pkg/tui/table.go:179]: Let's use standard `CTRL + R` / `F5` hotkeys for this

```suggestion
			if ev.Key() == tcell.KeyCtrlR || ev.Key() == tcell.KeyF5 {
```

**Review by ankitpokhrel — COMMENTED** [review]
Hi @GZLiew, Thank you for working on this 🙌🏻

I wanted to do this differently but that would require some rework on how the TUI is implemented. So, I think your approach is fine for now and I will improve it later.

I do have some comments tho.
1. Let's use standard `CTRL + R` / `F5` as a shortcut for refreshing.
2. Could you please extend this to `jira epic list <epic_id>` and `jira sprint list <sprint_id>`?

Thank you!

File: pkg/tui/table.go (hunk 1 above)
Comment (GZLiew) [reply pkg/tui/table.go:179]: Thanks for reviewing @ankitpokhrel ! alright I will proceed make a few more commits to address your comments above 👍🏻

File: README.md [hunk 2]
```diff
@@ -138,6 +138,7 @@ The lists are displayed in an interactive UI by default.
 - Use arrow keys or `j, k, h, l` characters to navigate through the list.
 - Use `g` and `SHIFT+G` to quickly navigate to the top and bottom respectively.
 - Press `v` to view selected issue details.
+- Press `r` to refresh issues list.
```
Comment (adolsalamanca) [inline README.md:141]: Nitpick, this is still pointing to previous refresh shortcut.

File: pkg/tui/table.go [hunk 3]
```diff
@@ -21,6 +21,9 @@ type SelectedFunc func(row, column int, data interface{})
 // ViewModeFunc sets view mode handler func which gets triggered when a user press 'v'.
 type ViewModeFunc func(row, col int, data interface{}) (func() interface{}, func(data interface{}) (string, error))
 
+// RefreshFunc is fired when a user press 'r' character in the table.
```
Comment (adolsalamanca) [inline pkg/tui/table.go:24]: this comment should also be updated

File: pkg/tui/table.go [hunk 4]
```diff
@@ -115,6 +119,13 @@ func WithViewModeFunc(fn ViewModeFunc) TableOption {
 	}
 }
 
+// WithRefreshFunc sets a func that is triggered when a user press 'r'.
```
Comment (adolsalamanca) [inline pkg/tui/table.go:122]: Another one that requires update

**Review by adolsalamanca — COMMENTED** [review]
Thanks for the work 👍🏽 (just another random contributor here) 
Could you please update Readme and comments so they stay up to date with latest shortcuts used?

File: README.md (hunk 2 above)
Comment (GZLiew) [reply README.md]: yes of course! Sorry I missed that!

File: pkg/tui/table.go (hunk 4 above)
Comment (GZLiew) [reply pkg/tui/table.go]: thanks for pointing it out for me @adolsalamanca

**Review by ankitpokhrel — APPROVED** [review]
Thank you for working on this @GZLiew and thank you for reviewing @adolsalamanca, it helps! 🎉

**Pull request merged**